doc/TSCC.rst

```diff
@@ -114,8 +207,12 @@ Notes:
 * Don't request more than one node per job. That means you would be managing inter-node inter-process communication yourself. (e.g. message 
   passing). Instead, just submit more jobs
 * If :code:`<log_dir>` is mistyped, the job will not run. Double check that location before you submit.
+* There may be an optional shebang line at the start of the file, but no blank or other lines between the beginning and the :code:`#SBATCH` lines
 * None of the SLURM settings can access environment variables. If you want to set a value (e.g. the log directory) dynamically, you'll
   need to dynamically generate the SLURM file.
+* SLURM does not support using environment variables in :code:`#SBATCH` lines in scripts. If you wish to use
+  environment variables to set such values, you must pass them to the :code:`sbatch` command directly
+  (e.g. :code:`sbatch --output=$SOMEWHERE/out slurm_script.sh`) 
```

Comment:
I didn't quite understand what this command sets

Reply:
```suggestion
  (e.g. :code:`sbatch --output=$SOMEWHERE/out slurm_script.sh`). In this case, the :code:`--output` command-line parameter will override the :code:`--output` in the header of the script.
```